interface/odmr_counter_interface.py

```diff
@@ -102,4 +106,32 @@ def get_odmr_channels(self):
 
         @return list(str): channels recorded during ODMR measurement
         """
-        pass
\ No newline at end of file
+        pass
+
+    @property
```

Comment:
Abstract methods only.
```
@property 
@abstractmethod 
```

should work according to documentation https://docs.python.org/3/library/abc.html#abc.abstractmethod

Reply:
that works.